samples/Microsoft.Azure.Storage.DataMovement.Client/CommandLineOptions.cs

```diff
@@ -20,6 +20,9 @@ internal class CommandLineOptions
         [Option('n', "transfers-number", HelpText = "Number of simultaneous transfers", Required = false, Default = 1)]
         public int TransfersNumber { get; set; }
 
+        [Option('i', "job-id", HelpText = "JobId of the transfer (e.g. to resume it later).", Required = false)]
```

Comment:
```suggestion
        [Option('j', "job-id", HelpText = "JobId of the transfer (e.g. to resume it later).", Required = false)]
```

`-i` is a common Unix option shortcut for an "interactive" mode. So I propose `-j`, but it's not something I will fight for relentlessly ;)

Reply:
I have changed it to `-j` 👍